q2_types/per_sample_sequences/__init__.py

```diff
@@ -18,6 +18,10 @@
                       SingleEndFastqManifestPhred64,
                       PairedEndFastqManifestPhred33,
                       PairedEndFastqManifestPhred64,
+                      SingleEndFastqManifestPhred33V2,
```

Comment:
I've been scratching my head over a better name for these formats, but I don't really have one. This does describe what they are...

Reply:
`SingleEndFastqManifestPhred33V2` -> `TheFormatFormerlyKnownAsSingleEndFastqManifestPhred33ButNowItIsMetadataCompatibleFormat` ?